.gitignore

```
@@ -22,9 +22,14 @@ xcuserdata/
 *.xccheckout
 *.xcscmblueprint
 
```

Could I get you to add an ignore for `.DS_Store` too, please?

```suggestion
.DS_Store
```